### DISCUSSION TITLE
Add quick delete button for automations

### DISCUSSION THREAD

**Discussion by Secretowo** [home-assistant/frontend#13279] [Automations / scripts / scenes] [answered]
I'd love to have a quick delete button in the automation list, since having to open the editor and waiting for it to load just to delete an automation can get annoying quickly.

Heres a mockup (proudly made in 1 minute using paint):
![image](https://user-images.githubusercontent.com/63597541/181041785-c2fdf2c1-9927-4615-a39d-2eca2c61a271.png)

**Comment by matthiasdebaat** [comment, marked as answer]
Love your Paint job! We should add this indeed.

We've made a prototype to make our tables consistent and more user friendly, especially on mobile, I would suggest to make the row clickable. For example like this [desktop (prototype)](https://www.figma.com/proto/FxvBgKik8oqwnjMx8PWQmc/Tables?page-id=1%3A1787&node-id=1%3A1823&viewport=900%2C531%2C0.45&scaling=min-zoom&starting-point-node-id=1%3A1823) and [mobile (prototype)](https://www.figma.com/proto/FxvBgKik8oqwnjMx8PWQmc/Tables?page-id=1%3A1787&node-id=1%3A1849&viewport=900%2C531%2C0.45&scaling=min-zoom&starting-point-node-id=1%3A1849&show-proto-sidebar=1).

Issue can be found here https://github.com/home-assistant/frontend/issues/12672
